packages/grant-explorer/src/features/discovery/__tests__/ApplyNowSection.test.tsx

```diff
@@ -28,7 +28,8 @@ vi.mock("common", async () => {
   };
 });
 
-describe("<ApplyNowSection />", () => {
+// This component is no longer used
+describe.skip("<ApplyNowSection />", () => {
```

Comment:
If it's not used then we can delete it - it's in git if we ever want to bring it back.

Reply:
Good point, I've removed them.